docs_src/phases/phase3.md

```diff
@@ -60,13 +60,88 @@ which will create the following docker image:
 
 This make target will clone the VAS repo and run the appropriate scripts to create the docker image.
 
-Run the following command to start the VAP services:
+### Environment variable section of docker-compose.vap.yml
+
+``` yaml
+environment:
+  - MQTT_DESTINATION_HOST=mqtt:1883
+  - CAMERA0_ENDPOINT=http://video-analytic:8080/pipelines/object_detection_cpu_render/1
+  # Fill in camera source info
+  - CAMERA0_SRC=rtsp://{URL}:{PORT}/{STREAM-END-POINT}
+  - CAMERA0_ROI_NAME=Staging
+  - CAMERA0_CROP_TBLR=0,0,0,0
+```
+
+- MQTT_DESTINATION_HOST - the destination of the MQTT broker, to use the MQTT broker running in the compose file simply just use `mqtt:1883` 
+- CAMERA{i}_ENDPOINT - the destination of the pipeline resource you wish to run the analytics on. Use [http://video-analytic:8080](http://video-analytic:8080/) to hit the container running the VAS software then just add the route to the pipeline you are targeting. 
+- CAMERA{i}_SRC - the path to the camera either RTSP or the Linux path to the camera resource 
+- CAMERA{i}_ROI_NAME - the name of the ROI location this camera represents, this is a configurable parameter of the reconciler also and these ROI locations will need to match what is defined there as well. 
+- CAMERA{i}_CROP_TBLR - the camera’s crop number of pixels to cut from the top, bottom, left, right. Try and crop the camera to just include the ROI of interest.
+
+Each of the CAMERA{i}_ environment variables can be incremented up to the number of cameras you wish to include in your solution.  For example, if you have two cameras your environment variables section would have the following values: 
+
+- MQTT_DESTINATION_HOST
+- CAMERA0_ENDPOINT
+- CAMERA0_SRC
+- CAMERA0_ROI_NAME
+- CAMERA0_CROP_TBLR
+- CAMERA1_ENDPOINT
+- CAMERA1_SRC
+- CAMERA1_ROI_NAME
+- CAMERA1_CROP_TBLR
+
+### Camera Source
+Before you run Video Analytics Serving you'll need to specify your camera source in the docker compose file `docker-compose.vap.yml`.
+
+Possible camera sources include:
+
+- rtsp
+- usb camera
+- local file (debugging only)
+
+#### RTSP
+
+For an RTSP feed edit the CAMERA0_SRC environment variable to point to the URL of the RTSP feed.
+
+`CAMERA0_SRC=rtsp://{URL}:{PORT}/{STREAM-END-POINT}`
+
+If the RTSP camera has username and password auth append those values to the beginning of the URL. i.e. `CAMERA0_SRC=rtsp://{USERNAME}:{PASSWORD}@{URL}:{PORT}/{STREAM-END-POINT}`.
+
+#### USB camera
+First, add the camera device to the `video-analytic` container.
+
+``` yaml
+devices:
+    - /dev/video0:/dev/video0
+```
+
+Then edit the CAMERA0_SRC environment variable to point to the video source
+`CAMERA0_SRC=/dev/video0`
+
+#### Local video file
+
+To use a local video sample add the path to the video file in the container to the CAMERA0_SRC environment variable.  The keyword `file` lets VAS know the URI is a local file and not a RTSP URL.
```

Comment:
do we have requirement about what kind of format (eg. mp4, wma, ...etc) for this video file? if so, probably want to list them out.